lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -16,6 +16,26 @@
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- These are state machine model tests for the DBLayer implementations.
```

Comment:
For Haddock comments, if you write `'DBLayer'` (in single quotes), then Haddock will turn this into a hyperlink that jumps to the `DBLayer` type definition.

Reply:
:+1: Fixed.